var/spack/repos/builtin/packages/bml/package.py

```diff
@@ -38,12 +38,24 @@ class Bml(CMakePackage):
     version('develop', git='https://github.com/lanl/bml', branch='master')
 
     variant('shared', default=True, description='Build shared libs')
+    variant('mpi', default=False, description='Build with MPI Support')
+
+    conflicts('+mpi', when='@:1.2.2')
+    conflicts('+mpi', when='@:1.1.0')
```

Comment:
line above already says "all versions lower or equal to 1.2.2". You don't need this one.